xwiki-platform-core/xwiki-platform-user/xwiki-platform-user-profile/xwiki-platform-user-profile-ui/src/main/resources/XWiki/XWikiUserSheet.xml

```diff
@@ -661,18 +667,20 @@ span#avatarUpload {
   margin: 0;
 }
 
-.attachment-picker-start {
-  background: url("$xwiki.getSkinFile('icons/silk/picture_edit.png')") no-repeat center center $theme.pageContentBackgroundColor !important;
-  border: 0 none !important;
-  border-bottom-left-radius: 8px;
-  height: 18px;
-  position: absolute;
-  right: 0;
-  text-align: left;
-  text-indent: -9999px;
-  top: 1px;
-  width: 18px !important;
-  z-index: 1;
+.attachment-picker {
+  .buttonwrapper {
+    .wikilink{
```

Comment:
We should add a space here to keep consistency in the codestyle:

> put a space between selector and declaration start, ex. "a {}"

from https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/XhtmlCssCodeStyle/

Reply:
So, here each rule also need to have a newline between them?

Reply:
Also, seeing the rest of this file, a lot of rules are not being followed, should I update the rest of the file as well?

Reply:
> So, here each rule also need to have a newline between them?

Those are not next to each other but nested so it's not mandatory. As far as I know we usually don't do it for nested rulesets.

Reply:
> Also, seeing the rest of this file, a lot of rules are not being followed, should I update the rest of the file as well?

From what I remember, it's okay to update codestyle in a file you're already editing.
If you want to do it, go for it, but there's no requirement as long as it's not code you've updated :)

Reply:
I adjusted my code with [df059ae](https://github.com/xwiki/xwiki-platform/pull/2862/commits/df059aeee9b7d17c145f390e3938f642e0bc803b) . I will do it for the rest of the file, but on a separate commit.